app/code/Magento/Backup/Model/ResourceModel/Table/GetListTables.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Backup\Model\ResourceModel\Table;
+
+use Magento\Framework\App\ResourceConnection;
+
+/**
+ * Class GetListTables
+ */
+class GetListTables
+{
+    const TABLE_TYPE = 'BASE TABLE';
+
+    /**
+     * @var ResourceConnection
+     */
+    private $resource;
+
+    /**
+     * @param ResourceConnection $resource
+     */
+    public function __construct(ResourceConnection $resource)
+    {
+        $this->resource = $resource;
+    }
+
+    /**
+     * @return array
+     */
+    public function execute()
+    {
+        return $this->resource->getConnection('backup')->fetchCol(
+"SHOW FULL TABLES WHERE `Table_type` = ?",
+            self::TABLE_TYPE
+            self::TABLE_TYPE
```

Comment:
Duplicate constant... Delete one please

Reply:
looks like github's suggested change doesn't work properly...

Reply:
Is a BETA feature. Maybe sometimes not works properly. Sorry